tests/schemas/Person-broken.avsc

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Person",
+  "type": "record",
+  "fields": [
+    {"name": "name", "type": "string", "example": "Donkey"},
+    {"name": "age", "type": ["null", "int"], "default": null, "example": "123"},
```

Comment:
I have one question. If types for that field are `["null", "int"]`, so example should be written as number, not as string, am I right - at least in JSON Schema we should write it in this way, is it different with avro?

Reply:
That line came from the existing example at https://github.com/asyncapi/avro-schema-parser/blob/61cbd19f972b1a7cbc8daf3f137c40e1d26aeaa5/tests/schemas/Person-1.9.0.avsc#L6 

but excuses-aside, yes - I think you're right - that should be an int, like `"example": 123` 

This needs fixing in:
- https://github.com/asyncapi/avro-schema-parser/blob/master/tests/schemas/Person-1.9.0-additional-attributes.avsc#L9 
- https://github.com/asyncapi/avro-schema-parser/blob/master/tests/schemas/Person-1.9.0-namespace.avsc#L7
- https://github.com/asyncapi/avro-schema-parser/blob/master/tests/schemas/Person-1.9.0.avsc#L6

Reply:
I wonder if someone made a mistake then and it needs to be corrected in all places. It would be best to ask someone who is an expert in avro. As far as I'm concerned this is the only thing that caught my eye, the rest is ok :)

Reply:
this is right but because the `example` attribute is not an official AVRO attribute, this is not critical and i guess that the validation should pass if the example value is not correctly typed (because it is an additional attribute ...)

Reply:
created https://github.com/asyncapi/avro-schema-parser/issues/97 for this